src/ui/molecules/pricing-table.svelte

```diff
@@ -45,7 +45,9 @@
             {$translator.translate(LocalizationKeys.PricingTableTax)}
           </div>
           <div class="rcb-pricing-table-value">
-            <div class="rcb-pricing-table-value-loading">Loading</div>
+            <div class="rcb-pricing-table-value-loading">
+              {$translator.formatPrice(12340000, priceBreakdown.currency)}
```

Comment:
I guess this is a placeholder for the loading?

Reply:
Yes, I put an arbitrary number of 4 digits without any extra padding `$12.34`. Initially tried `$1.23` but it felt too narrow visually. I implemented it this way in case some currencies take more than a symbol and more width, e.g.: swedish crowns I assume are represented as `12.34 SEK`.